manager/dispatcher/telegram.go

```diff
@@ -112,6 +113,13 @@ Plugin Name: <em>%s</em>
 		return err
 	}
 	defer response.Body.Close()
+
+	if response.StatusCode < 200 || response.StatusCode > 299 {
+		log.Error().Str("module", "dispatcher").Msgf("Channel(Telegram): Error in Response with Error code: %d", response.StatusCode)
+		errorMessage := fmt.Sprintf("REST API Error with HTTP response status code: %d", response.StatusCode)
```

Comment:
Suggest to use `fmt.Errorf` function to create error instance.

Reply:
I've updated as you mentioned. Can you review again?

Reply:
awesome thanks for considering.